merlin/systems/dag/ops/workflow.py

```diff
@@ -91,7 +91,7 @@ def export(self, path, input_schema, output_schema, node_id=None, version=1):
             modified_workflow,
             node_name,
             node_export_path,
-            backend="python",
+            backend="nvtabular",
```

Comment:
Will this work on the containers? I saw this PR go through, and I'm wondering if the NVT backend is still available https://github.com/NVIDIA-Merlin/Merlin/pull/378/files

Reply:
thats why I need the other PR in merlin to go through also... need to recook